components/organisms/ContributorProfileTab/contributor-profile-tab.tsx

```diff
@@ -371,8 +371,8 @@ const ContributorProfileTab = ({
             </div>
             <div className="grid grid-cols-2 2xl:grid-cols-4 justify-between gap-2 lg:flex-row md:gap-12 lg:gap-16">
               <div>
-                <span className="flex gap-2 text-xs text-light-slate-11">
-                  OSCR Rating
+                <span className="flex gap-2 text-sm">
```

Comment:
The info icon is larger now, but the text ends up being misaligned now.

![CleanShot 2024-08-05 at 15 20 25](https://github.com/user-attachments/assets/a0f44182-1223-49e4-84c7-de471183a4f7)

![CleanShot 2024-08-05 at 15 20 06](https://github.com/user-attachments/assets/b3366341-7349-4eba-97ed-4079bd1ec441)

Reply:
It's still off, but I figured a way to have it bigger and line up properly.

For the OSCR set the JSX to:

```jsx
              <div>
                <span className="relative text-xs text-light-slate-11 flex gap-0.5 items-center">
                  <span>OSCR Rating</span>
                  <span className="text-sm grid place-content-center">
                    <InfoTooltip information="OSCR evaluates the engagement and impact of contributors across the entire open source ecosystem." />
                  </span>
                </span>
                <div className="flex mt-1 text-lg md:text-xl lg:text-2xl !text-black leading-none">
                  <OscrButton rating={contributor?.oscr} hideRating={!Boolean(user)} />
                </div>
              </div>
```

And for the other stats, add `flex-items center`, e.g.

```diff
              <div>
-                <span className="text-xs text-light-slate-11">PRs opened</span>
+                <span className="text-xs text-light-slate-11 flex items-center">PRs opened</span>

                {totalPrs ? (
                  <div className="flex mt-1 !text-lg md:!text-xl lg:!text-2xl !text-black !leading-none">
                    {totalPrs} PRs
                  </div>
                ) : (
                  <div className="mt-1">{DATA_FALLBACK_VALUE}</div>
                )}
              </div>
```